pkgs/applications/office/sozi/default.nix

```diff
@@ -0,0 +1,38 @@
+{ appimageTools, lib, fetchurl }:
```

Comment:
Please move this file to `pkgs/by-name/so/sozi/package.nix`; see [`pkgs/by-name/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md) for details.